### PR TITLE
[MM-18700] UI Automation: "Leave a long draft in reply input box"

### PR DESCRIPTION
**Pull request by kosgrz**
**Summary**
Write an automated test using Cypress for "Leave a long draft in reply input box"

**Ticket Link**
Fixes https://github.com/mattermost/mattermost-server/issues/12288

### CODE REVIEW

File: e2e/cypress/integration/messaging/post_textbox_height_spec.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        getInputBox().should('have.css', 'height', '46px');
+
+        // # Write a long text in input
+        getInputBox().type('test\n\n\n\n\n\n');
+
+        // # Check that input box is taller than before
+        getInputBox().should('have.css', 'height', '166px');
+
+        // # Go to another channel
+        cy.get('#sidebarItem_suscipit-4').click();
```
Comment (reflog) [inline e2e/cypress/integration/messaging/post_textbox_height_spec.js]: can we avoid depending on channel name here? test will fail if channel is named differently...

Comment (kosgrz) [reply e2e/cypress/integration/messaging/post_textbox_height_spec.js]: Thanks, I click "resolve" as I don't need to go to another channel finally :)

**Comment by kosgrz** [comment]
@srkgupta oh, sorry for a mistake. I will send another commit in this situation :)

File: e2e/cypress/integration/messaging/post_textbox_height_spec.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and navigate to town-square
+        cy.toMainChannelView('user-1');
+
+        // # Add two posts
+        getInputBox().type('test post 1{enter}');
+        getInputBox().type('test post 2{enter}');
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/post_textbox_height_spec.js]: nit: We can use existing custom command `cy.postMessage` that almost doing the same thing.
```javascript
        cy.postMessage('test post 1');
        cy.postMessage('test post 2');
```

Comment (kosgrz) [reply e2e/cypress/integration/messaging/post_textbox_height_spec.js]: Done :+1:

File: e2e/cypress/integration/messaging/post_textbox_height_spec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        // # Get latest post id
+        cy.getLastPostId().then((latestPostId) => {
+            // # Click reply icon
+            cy.clickPostCommentIcon(latestPostId);
+
+            // # Make sure that text box has initial height
+            getTextBox().should('have.css', 'height', '100px');
+
+            // // # Write a long text in text box
+            getTextBox().type('test\n\n\n\n\n\n');
```
Comment (saturninoabril) [inline e2e/cypress/integration/messaging/post_textbox_height_spec.js]: This is not getting posted in recent Cypress version.  Please change to,
```suggestion
            getTextBox().type('test{shift}{enter}{enter}{enter}{enter}{enter}{enter}');
```

Comment (kosgrz) [reply e2e/cypress/integration/messaging/post_textbox_height_spec.js]: Done :+1:

**Pull request merged**